src/commands/protocols/common/epoch-check-command.js

```diff
@@ -148,7 +148,7 @@ class EpochCheckCommand extends Command {
                 r2,
                 serviceAgreement.hashFunctionId,
                 serviceAgreement.scoreFunctionId,
-                true,
+                false,
```

Comment:
Why are we not filtering anymore last seen nodes?

Reply:
We use these nodes to calculate left, center and right edge of the neighbourhood for commits and proofs. This calculation needs to be the same as it is on blockchain, and blockchain doesn't care if node was unavailable for some time on the network.

Reply:
If a node is not seen for some time, he is still in the sharding table on chain, and has an index, we calculate indexes on the node from the whole sharding table, if get all is missing some offline nodes it will skew indexes.

Reply:
Thanks for explanation guys